src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck started working.");
```

Comment:
The message 'Truck started working.' could be more informative. According to the checklist, it should indicate the type of machine that is working. Consider including the specific action the Truck is performing, for example, 'Truck is hauling cargo.'